CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Version 0.1.0 - tbd
```

Comment:
tbd -> today (or remove)

Reply:
I thought we would do that in the merge from `main` to the `release` branch, but we could also do it now. What do you think?

Reply:
Yes, let's do that when we know the date.